Example/CollectionViewToolsExample/MainViewController.swift

```diff
@@ -389,6 +391,26 @@ class MainViewController: UIViewController {
         }
     }
     
+    // MARK: Change images
+    
+    func makeChangeImagesActionCellItem() -> CollectionViewCellItem {
+        return makeActionCellItem(title: "Change images") { [weak self] in
+            guard let `self` = self, self.images.isEmpty == false else {
+                return
+            }
+            
+            self.mainCollectionViewManager.sectionItems.forEach { sectionItem in
+                sectionItem.cellItems.forEach { cellItem in
+                    guard let cellItem = cellItem as? ImageCellItem,
+                        let cell = cellItem.cell as? ImageCollectionViewCell else {
```

Comment:
Should you check `cellItem as? ImageCellItem` here of check cell class is enough?